src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle() {
+        this.radius = getNumber();
+    }
```

Comment:
we shouldn't hardcode values in the constructor
```suggestion
    public Circle(String color, int radius) {
        ...
    }
```